### PR TITLE
[master] Porting #52838 to master

### PR DESCRIPTION
**Pull request by garethgreenaway**
Porting #52838 to master

### CODE REVIEW

File: salt/modules/win_psget.py [hunk 1]
```diff
@@ -50,14 +49,42 @@ def __virtual__():
     return __virtualname__
 
 
-def _pshell(cmd, cwd=None, json_depth=2):
+def _ps_xml_to_dict(parent, dic=None):
+    '''
+    Formats powershell Xml to a dict.
+    Note: This _ps_xml_to_dict is not perfect with powershell Xml.
```
Comment (dwoz) [inline salt/modules/win_psget.py]: @cmcmarrow 
What does this mean `is not perfect with powershell Xml`?

Comment (Ch3LL) [reply salt/modules/win_psget.py]: bump @cmcmarrow

**Comment by Ch3LL** [comment]
Closing since this was included in the 2018.3 merge forward here https://github.com/saltstack/salt/pull/55027

File: salt/modules/win_psget.py (hunk 1 above)
Comment (cmcmarrow) [reply salt/modules/win_psget.py]: XML and Dictionaries are not one to one. But in this uses cases it is. We should be using Json here but at the time there was a upstream bug in with the PowerShell program.

Comment (cmcmarrow) [reply salt/modules/win_psget.py]: @dwoz

**Pull request closed without merging**